data/dataset_definitions.yml

```diff
@@ -1,4 +1,11 @@
 datasets:
+  - name: HumanMattingClips120
+    annotation_conversion:
+      converter: background_matting_sequential
```

Comment:
Please add required arguments: masks_dir and images_dir

Reply:
Should not other optional arguments be specified?

Reply:
should, if they differ from the default arguments

Reply:
Updated with all necessary parameters.